Week2/homework/js-exercises/readinglist-ex4.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+let books = [
+    {
+        title: "The Three Musketeers",
+        author: "Alexandre Dumas",
+        alreadyRead: true,
+    },
+    {
+        title: "Harry Potter",
+        author: "J.K Rolling",
+        alreadyRead: false,
+    },
+    {
+        title: "Deep Work",
+        author: "Kal Newport",
+        alreadyRead: false,
+    }
+]
+
+
+
+books.forEach((element, index, array) => {
+    console.log(element.title + " by " + element.author);
+    if (element.alreadyRead === true) {
```

Comment:
For a boolean you don't need `===`. simply say `if (element.alreadyRead)`

Reply:
resolved